src/Compilers/VisualBasic/Portable/Symbols/MissingMetadataTypeSymbol.vb

```diff
@@ -167,23 +172,50 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 End Get
             End Property
 
-            Public Overrides ReadOnly Property SpecialType As SpecialType
+            Private ReadOnly Property TypeId As Integer
```

Comment:
The `SpecialType` property is still strongly-typed, as before. The `TypeId` property is just factoring some code out.